### PR TITLE
Adding documentation about the simple cache PSR-16 implementation

### PR DESCRIPTION
**Pull request by weaverryan**
Hi guys!

This covers #7409!

Cheers!

### CODE REVIEW

File: components/cache.rst [hunk 1]
```diff
@@ -21,11 +25,27 @@ You can install the component in 2 different ways:
 * :doc:`Install it via Composer </components/using_components>` (``symfony/cache`` on `Packagist`_);
 * Use the official Git repository (https://github.com/symfony/cache).
 
-Key Concepts
-------------
+Cache (PSR-6) Versus Simple Cache (PSR-16)
+------------------------------------------
```
Comment (weaverryan) [inline components/cache.rst]: @nicolas-grekas Is this small section accurate? Everything else is straightforward - I just wanted you to check this one spot! Thanks!

File: components/cache.rst [hunk 2]
```diff
@@ -71,14 +91,98 @@ Now you can create, retrieve, update and delete items using this cache pool::
...
+    ]);
+
+Available Simple Cache (PSR-16) Classes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following cache adapters are available:
+
+.. tip::
+
+    To find out more about each of these classes, you can read th
```
Comment (nicolas-grekas) [inline components/cache.rst]: you can read theee

**Review by nicolas-grekas — COMMENTED** [review]
LGTM, thanks!

File: components/cache/cache_pools.rst [hunk 3]
```diff
@@ -156,6 +197,25 @@ where it was missing. Since it's not possible to know the expiry date and time
...
+This adapter wraps another adapter, and allows you to read a report of all of the
+"calls" made to the adapter, including how long actions took, cache hits, misses
+and more::
+
+    use Symfony\Component\Cache\Adapter\TraceableAdapter;
+    use Symfony\Component\Cache\Adapter\FilesystemAdapter;
+
+    $fileCache = new FilesystemAdapter();
+
+    $cache = new TraceableAdapter(array($fileCache));
```
Comment (nicolas-grekas) [inline components/cache/cache_pools.rst]: No array wrapper

File: components/cache.rst [hunk 4]
```diff
@@ -71,14 +91,98 @@ Now you can create, retrieve, update and delete items using this cache pool::
...
 
 .. toctree::
     :glob:
     :maxdepth: 1
 
     cache/*
 
+.. _cache-component-psr16-caching:
+
+Simple Caching (PSR-16)
```
Comment (javiereguiluz) [inline components/cache.rst]: I'd move the simple cache part (PSF-16) before the complex cache part (PSR-6). That's the usual practice in docs: simple first and then build up to complex.

File: components/cache.rst [hunk 5]
```diff
@@ -71,14 +91,98 @@ Now you can create, retrieve, update and delete items using this cache pool::
...
+-----------------------
+
+This part of the component is an implementation of `PSR-16`_, which means that its
+basic API is the same as defined in the standard. First, create a cache object from
+one of the built-in cache clases. For example, to create a filesystem-based cache,
```
Comment (javiereguiluz) [inline components/cache.rst]: `clases` -> `classes`

File: components/cache/cache_pools.rst [hunk 6]
```diff
@@ -5,8 +5,8 @@
    single: Redis Cache
    single: PDO Cache, Doctrine DBAL Cache
 
-Cache Pools
-===========
+Cache Pools & Supported Adapters
```
Comment (javiereguiluz) [inline components/cache/cache_pools.rst]: `&` -> `and` ?

File: components/cache/cache_pools.rst [hunk 7]
```diff
@@ -108,6 +109,46 @@ helper allows creating a connection to a Redis server using a DSN configuration:
...
+    );
+
+The :method:`Symfony\\Component\\Cache\\Adapter\\Memcached::createConnection`
+helper allows creating a connection to a pool of Memcached server using a DSN configuration::
+
+    $memcachedClient = MemcachedAdapter::createConnection(
+        'memcached://user:pass@localhost?weight=33',
+
+        // options, including username, password and Memcached::OPT_* options
+        ['persistent_id' => '_products_cache']
```
Comment (javiereguiluz) [inline components/cache/cache_pools.rst]: Long array syntax here.

File: components/cache/cache_pools.rst [hunk 8]
```diff
@@ -108,6 +109,46 @@ helper allows creating a connection to a Redis server using a DSN configuration:
...
+    );
+    $memcachedClient = MemcachedAdapter::createConnection([
```
Comment (javiereguiluz) [inline components/cache/cache_pools.rst]: And here too, long array syntax

**Review by javiereguiluz — CHANGES_REQUESTED** [review]
Thanks for adding this!

**Comment by weaverryan** [comment]
Feedback applied! And I added the last section about converting between the 2 caches

Status: Needs Review

File: components/cache.rst [hunk 9]
```diff
@@ -21,11 +25,113 @@ You can install the component in 2 different ways:
...
+    }
+
+    // retrieve the value stored by the item
+    $numProducts = $cache->get('stats.num_products');
+
+    // or specify a default value, if the key doesn't exist
+    // $numProducts = $cache->get('stats.num_products', 100);
+
+    // remove the cache key
+    $cache->deleteItem('stats.num_products');
```
Comment (javiereguiluz) [inline components/cache.rst]: `deleteItem()` should be `delete()` ?

File: components/cache/cache_pools.rst [hunk 10]
```diff
@@ -108,6 +109,46 @@ helper allows creating a connection to a Redis server using a DSN configuration:
...
+The :method:`Symfony\\Component\\Cache\\Adapter\\Memcached::createConnection`
+helper allows creating a connection to a pool of Memcached server using a DSN configuration::
+
+    $memcachedClient = MemcachedAdapter::createConnection(
+        'memcached://user:pass@localhost?weight=33',
+
+        // options, including username, password and Memcached::OPT_* options
+        array('persistent_id' => '_products_cache')
+    );
+    $memcachedClient = MemcachedAdapter::createConnection(array(
```
Comment (javiereguiluz) [inline components/cache/cache_pools.rst]: Is this repeated `$memcachedClient` variable defined on purpose? Should we merge its contents with the previous variable?

**Review by javiereguiluz — APPROVED** [review]
👍 

Status: reviewed

**Pull request closed without merging**